npm-shrinkwrap.json

```diff
@@ -2,7 +2,7 @@
   "name": "marathon-ui",
   "version": "0.9.0",
   "npm-shrinkwrap-version": "5.4.0",
-  "node-version": "v0.10.38",
```

Comment:
Someone's using Node v0.10. I don't think this will cause many problems, but it will cause unnecessary noise in the shrinkwrap file as module versions go up and down. Travis CI uses 0.12.*, we should too.